pretyping/detyping.ml

```diff
@@ -432,7 +434,8 @@ let detype_anonymous = ref (fun loc n -> anomaly ~label:"detype" (Pp.str "index
 let set_detype_anonymous f = detype_anonymous := f
 
 let detype_level sigma l =
-  GType (Some (dl, Pp.string_of_ppcmds (Evd.pr_evd_level sigma l)))
+  let l = Pp.string_of_ppcmds (Evd.pr_evd_level sigma l) in
+  GType (Some (dl, Name.mk_name (Id.of_string l)))
```

Comment:
Invalid identifier is only a warning which is disabled by default, so it didn't appear.
I'll change it to Id.of_string_soft which only fails on empty string and invalid unicode.